Makefile

```diff
@@ -1,29 +1,37 @@
-.PHONY: lint test install build build-dleq mock
```

Comment:
These were out of sync with the rest of the file. It's uglier, but putting the `.PHONY` label above each non-filesystem target is the only way I've found to correctly maintain them.